src/libraries/System.Net.Quic/tests/FunctionalTests/MsQuicTestBase.cs

```diff
@@ -7,13 +7,16 @@
 
 namespace System.Net.Quic.Tests
 {
+    // TODO: why do we hawe 2 base clase with some duplicated methods?
     public class MsQuicTestBase
     {
         public SslServerAuthenticationOptions GetSslServerAuthenticationOptions()
         {
             return new SslServerAuthenticationOptions()
             {
-                ApplicationProtocols = new List<SslApplicationProtocol>() { new SslApplicationProtocol("quictest") }
+                ApplicationProtocols = new List<SslApplicationProtocol>() { new SslApplicationProtocol("quictest") },
+                // TODO: use a cert. MsQuic currently only allows certs that are trusted.
```

Comment:
What is the cause of this trusted limitation? Is it a requirement of QUIC, or because we haven't added configuration to ignore invalid certificates yet?

Reply:
we may need to plumb custom validation callback to managed code. I'm not sure if that was already done.

Reply:
https://github.com/dotnet/runtime/issues/50156